eduvpn/util.py

```diff
@@ -134,6 +135,25 @@ def have_dbus():
         return True
 
 
+#lru_cache(maxsize=1)
```

Comment:
looking good but i guess this comment can be removed.

Reply:
and also the pep8 check fails on this :) if you fix this I'll approve the merge.

Reply:
I just fixed the erroneous comment.

I guess the pep8 issue is related to excessive lengths for lines 144, 146 and 150 (I don't see anything else in 'pep8' tool output). As line wrapping is an highly cosmetic issue, I prefer to let you handle it with your own style preferences. BTW, those are not the only ones lines whose length exceed 79 characters, they are 11 others in the same file.